_includes/footer.html

```diff
@@ -1,7 +1,7 @@
 ﻿<footer class="content-info" role="contentinfo">
 
 <div class="container" style=" text-align: center">
-  <p>&copy; 2015-2018 OData – The Protocol for REST APIs </p>
+  <p>&copy; 2015-{{ 'now' | date: "%Y" }} OData – The Protocol for REST APIs </p>
```

Comment:
The assumption here is the site is updated at least yearly. Could be improved by having this calculation client side